shoplist/build.gradle

```diff
@@ -0,0 +1,59 @@
+plugins {
+    id 'com.android.library'
+    id 'kotlin-android'
+}
+
+android {
+    compileSdkVersion 30
+    buildToolsVersion "30.0.3"
+
+    defaultConfig {
+        minSdkVersion 21
+        targetSdkVersion 30
```

Comment:
you have to increase the targetSdkVersion, but remember to set the export value to true or false in your activities declared in the Manifest.xml file.

Reply:
Android version +30 like 31 or 32 requires to add the export attribute in your manifest file for each activity, service, or provider declared in that file.